UrbanDictionary/Controllers/WordsController.cs

```diff
@@ -76,5 +67,31 @@ public ActionResult Delete(long id)
             if (_serviceWrapper.Word.TryDelete(id)) return Ok(id);
             return NotFound(id);
         }
+
+        [HttpPut("approve/{id}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public ActionResult Approve(long id)
+        {
+            if (_serviceWrapper.Word.TryApproveWord(id)) return Ok(id);
+            return BadRequest(id);
+        }
+
+        [HttpPut("disapprove/{id}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status400BadRequest)]
+        public ActionResult Disapprove(long id)
+        {
+            if (_serviceWrapper.Word.TryDisapproveWord(id)) return Ok(id);
+            return BadRequest(id);
+        }
+
+        [HttpGet("getByTag/{tag}")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        public ActionResult<IEnumerable<WordDTO>> GetByTag(string tag)
+        {
+            IEnumerable<WordDTO> words = _serviceWrapper.Word.GetByTagName(tag);
+            return Ok(words);
```

Comment:
you can return like that: return Ok(_serviceWrapper.Word.GetByTagName(tag));

Reply:
I know, now it looks better.

Reply:
Fixed